core/sql/optimizer/RelExpr.cpp

```diff
@@ -11388,8 +11388,9 @@ void RelRoot::setMvBindContext(MvBindContext * pMvBindContext)
   pMvBindContextForScope_ = pMvBindContext;
 }
 
-void RelRoot::addOneRowAggregates(BindWA* bindWA)
+NABoolean RelRoot::addOneRowAggregates(BindWA* bindWA, NABoolean forceGroupByAgg)
 {
+  NABoolean GroupByAggNodeAdded = FALSE;
```

Comment:
A small nit: This variable should start with a lower case letter.

Reply:
Done; changed it to start with lower case letter